### DISCUSSION TITLE
Welcome to Mycroft-Cat Discussions!

### DISCUSSION THREAD

**Discussion by pazoff** [pazoff/Mycroft-Cat#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Suggest New Features
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BW-KING** [comment]
Mycroft and Local-Cat running in docker together have so much potential. The idea I have is to create a skill for Mycroft that whenever it does not know the answer or what to do (which is quite often - LOL), it automatically sends that request to the cat.

Secondly, there could be a Skill that the user can request to to automatically be connected to a back and forth chat session with the cat - Example - "Hey Mycroft - Lets Chat" 

I am sure there will need to be matching plugins in the cat to work with these Mycroft skills
Let me know what you think of these ideas
